docs/how-to/join-a-network.md

```diff
@@ -96,43 +96,84 @@ To see the latest network activity, including **chain block height**, **block he
 
 ## Upgrade in place
 Here’s how to upgrade in place, with minimum impact:
-* Back up your wallet, just in case: `lotus wallet export t3… > path/wallet.keyinfo`
-* Build the new binary.
-* If you are running miner(s), check if your miner is safe to shut down and restart: `lotus-miner proving info`. If any deadline shows a block height in the past, do not restart. See examples below.
-* Otherwise, you can safely restart your daemon and miner(s). This will replace and update the binary, and should preserve your config, wallets, and anything else on disk.
-
-In the following example, Deadline Open is 454 which is earlier than Current Epoch of 500. This miner should **not** be shut down or restarted.
-```
-$ sudo lotus-miner proving info
-Miner: t01001
-Current Epoch:           500
-Proving Period Boundary: 154
-Proving Period Start:    154 (2h53m0s ago)
-Next Period Start:       3034 (in 21h7m0s)
-Faults:      768 (100.00%)
-Recovering:  768
-Deadline Index:       5
-Deadline Sectors:     0
-Deadline Open:        454 (23m0s ago)
-Deadline Close:       514 (in 7m0s)
-Deadline Challenge:   434 (33m0s ago)
-Deadline FaultCutoff: 384 (58m0s ago)
-```
-
-In this next example, the miner can be safely restarted because no Deadlines are earlier than Current Epoch of 497. You have ~45 minutes before the miner must be back online to declare faults (FaultCutoff). If the miner has no faults, you have about an hour.
-```
-$ sudo lotus-miner proving info
-Miner: t01000
-Current Epoch:           497
-Proving Period Boundary: 658
-Proving Period Start:    658 (in 1h20m30s)
-Next Period Start:       3538 (in 25h20m30s)
-Faults:      0 (0.00%)
-Recovering:  0
-Deadline Index:       0
-Deadline Sectors:     768
-Deadline Open:        658 (in 1h20m30s)
-Deadline Close:       718 (in 1h50m30s)
-Deadline Challenge:   638 (in 1h10m30s)
-Deadline FaultCutoff: 588 (in 45m30s)
-```
+* `Back up your wallet`: just in case
+  ```bash
+  lotus wallet export t3… > path/wallet.keyinfo
+  ```
+* `Find a good upgrade window`: If you are running miner(s), check if your miner is safe to shut down and restart: 
+  ```bash
+  lotus-miner proving info
+  ```
+  If any deadline shows a block height in the past, do not restart. See examples below.
+
+    In the following example, `Deadline Sectors` is 0 which means no sectors are currently being challenged in this deadline. This deadline closes in 7 minutes, as indicated by `Deadline Close`. This means the miner can be safely restarted in the next 7 minutes, before the next deadline reaches.
```

Comment:
I don't think the bit about `7 minutes` is reliable here -- pretty sure Lotus is displaying a different value here. I'd take it out for the time being (see https://github.com/filecoin-project/lotus/issues/3630).

Reply:
updated